### ISSUE TITLE
So many broken symlinks exist in apollo repo.

### ISSUE THREAD

**Issue by hongyi-zhao** [ApolloAuto/apollo#12222]
Hi,

It seems there are so many broken symlinks exist in apollo repo. See my following testing for more info which is done under the root directory of the local git master repo of apollo:

```
$ find . -xtype l 
./modules/perception/testdata/lidar/lib/classifier/decision_forest/conf/decision_forest.config
./modules/perception/testdata/lidar/lib/classifier/decision_forest/data/perception/lidar/models/classifier/decision_forest
./modules/perception/camera/tools/offline/conf
./modules/perception/camera/tools/offline/data
./modules/perception/camera/tools/obstacle_detection/data/multicue
./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half-config.pt
./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half
./modules/perception/camera/tools/obstacle_detection/data/yolo/config.pt
./modules/dreamview/frontend/dist/assets/map_data
```

Best regards,
HY

**Comment by storypku** [comment]
> > Hi,
> > It seems there are so many broken symlinks exist in apollo repo. See my following testing for more info which is done under the root directory of the local git master repo of apollo:
> > ```
> > $ find . -xtype l 
> > ./modules/perception/testdata/lidar/lib/classifier/decision_forest/conf/decision_forest.config
> > ./modules/perception/testdata/lidar/lib/classifier/decision_forest/data/perception/lidar/models/classifier/decision_forest
> > ./modules/perception/camera/tools/offline/conf
> > ./modules/perception/camera/tools/offline/data
> > ./modules/perception/camera/tools/obstacle_detection/data/multicue
> > ./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half-config.pt
> > ./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half
> > ./modules/perception/camera/tools/obstacle_detection/data/yolo/config.pt
> > ./modules/dreamview/frontend/dist/assets/map_data
> > ```
> > 
> > 
> > Best regards,
> > HY
> 
> Thanks for reporting! We will evaluate and fix it ASAP.
> 
> cc also @jeroldchen @panfengsu

Use the following command to see more of these broken links.
```
find modules/ -xtype l -exec ls -lh --color=always {} +
```

@jeroldchen

**Comment by storypku** [comment]
> Hi,
> 
> It seems there are so many broken symlinks exist in apollo repo. See my following testing for more info which is done under the root directory of the local git master repo of apollo:
> 
> ```
> $ find . -xtype l 
> ./modules/perception/testdata/lidar/lib/classifier/decision_forest/conf/decision_forest.config
> ./modules/perception/testdata/lidar/lib/classifier/decision_forest/data/perception/lidar/models/classifier/decision_forest
> ./modules/perception/camera/tools/offline/conf
> ./modules/perception/camera/tools/offline/data
> ./modules/perception/camera/tools/obstacle_detection/data/multicue
> ./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half-config.pt
> ./modules/perception/camera/tools/obstacle_detection/data/yolo/3d-r4-half
> ./modules/perception/camera/tools/obstacle_detection/data/yolo/config.pt
> ./modules/dreamview/frontend/dist/assets/map_data
> ```
> 
> Best regards,
> HY

To be fixed in PR #12229
